code/modules/spells/spell_types/pointed/projectile/falcon_disrupt.dm

```diff
@@ -0,0 +1,213 @@
+/datum/action/cooldown/spell/projectile/falcon_disrupt
+	name = "Wings of Dendor"
+	desc = "Summons a falcon that could deliver small items and act as a distraction."
+	button_icon_state = "verd"
+	sound = 'sound/vo/mobs/bird/birdfly.ogg'
+	invocation = "The Treefather commands thee, TO FLY!"
+	invocation_type = INVOCATION_SHOUT
+	spell_type = SPELL_MIRACLE
+	antimagic_flags = MAGIC_RESISTANCE_HOLY
+	associated_skill = /datum/skill/magic/holy
+	required_items = list(/obj/item/clothing/neck/psycross/silver/dendor)
+	attunements = list(/datum/attunement/earth = 0.5)
+	charge_time = 1 SECONDS
+	charge_drain = 1
+	charge_slowdown = 0.7
+	cooldown_time = 50 SECONDS
+	spell_cost = 20
+	projectile_type = /obj/projectile/magic/falcon_dive
+
+/datum/action/cooldown/spell/projectile/falcon_disrupt/cast(atom/cast_on)
```

Comment:
Im not going to ask you to change it but having this as a projectile spell is weird